foreman/data_refinery_foreman/foreman/management/commands/remove_invalid_data.py

```diff
@@ -0,0 +1,50 @@
+
+from django.core.management.base import BaseCommand
+from data_refinery_common.models import (
+    Sample,
+    ComputationalResult,
+    ComputationalResultAnnotation,
+    Organism
+)
+from data_refinery_common.logging import get_and_configure_logger
+
+logger = get_and_configure_logger(__name__)
+
+# Minimum number of microarray samples required to create a QN Target
+QN_TARGET_SAMPLE_THRESHOLD = 100
```

Comment:
Would it be better to make this an argument to the command with 100 used as the default in case we one day change our mind on 100?

Reply:
sure, good idea!